### PR TITLE
add:`stockfish-git`

### PR DESCRIPTION
**Pull request by pythoncrazy**
Added stockfish-git

### CODE REVIEW

**Comment by pythoncrazy** [comment]
Sorry, I accidentally pushed the wrong version of the pacscript. I updated it just now

File: packages/stockfish-git/stockfish-git.pacscript [hunk 1]
```diff
@@ -0,0 +1,23 @@
+maintainer="pythoncrazy <encryptprogramming@gmail.com>"
+
```
Comment (Elsie19) [inline packages/stockfish-git/stockfish-git.pacscript:2]: Not at the top.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/stockfish-git/stockfish-git.pacscript (hunk 1 above)
Comment (pythoncrazy) [reply packages/stockfish-git/stockfish-git.pacscript:2]: Oops, I will change

File: packages/stockfish-git/stockfish-git.pacscript [hunk 2]
```diff
@@ -0,0 +1,22 @@
+name="stockfish-git"
+pkgname="stockfish-git"
```
Comment (Elsie19) [inline packages/stockfish-git/stockfish-git.pacscript:2]:
```suggestion
```

File: packages/stockfish-git/stockfish-git.pacscript [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name="stockfish-git"
+pkgdesc="The Stockfish chess engine, built from the git repository"
+homepage='https://stockfishchess.org/'
+url="https://github.com/official-stockfish/Stockfish.git"
+maintainer="pythoncrazy <encryptprogramming@gmail.com>"
+pkgver="0.0.1"
+gives="stockfish"
```
Comment (Elsie19) [inline packages/stockfish-git/stockfish-git.pacscript:7]: put this below name.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/stockfish-git/stockfish-git.pacscript (hunk 3 above)
Comment (pythoncrazy) [reply packages/stockfish-git/stockfish-git.pacscript]: Is my latest commit correct?

File: packages/stockfish-git/stockfish-git.pacscript [hunk 4]
```diff
@@ -0,0 +1,21 @@
+name="stockfish-git"
+gives="stockfish"
+pkgdesc="The Stockfish chess engine, built from the git repository"
```
Comment (Elsie19) [inline packages/stockfish-git/stockfish-git.pacscript:3]: don't include the part about it being from git.

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/stockfish-git/stockfish-git.pacscript (hunk 4 above)
Comment (pythoncrazy) [reply packages/stockfish-git/stockfish-git.pacscript:3]: okie dokie

Comment (pythoncrazy) [reply packages/stockfish-git/stockfish-git.pacscript]: Is this good

**Review by Elsie19 — APPROVED** [review]

**Comment by pythoncrazy** [comment]
God Bless!

**Pull request merged**